### PR TITLE
Istio Service Quickstart #917

### PR DESCRIPTION
**Pull request by shawnj**
# Summary

This is an Istio Services Quickstart. A dashboard for views into Services/Applications running within an Istio Service Mesh.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [x] Did you check that your alerts actually work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: quickstarts/istio/config.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Name of the quickstart (required)
+name: istio-quickstart
```
Comment (jpvajda) [inline quickstarts/istio/config.yml:2]: We've been recommending to not use the term "quickstart" in a quickstart as it's redundant.

Comment (jpvajda) [inline quickstarts/istio/config.yml:2]:
```suggestion
name: istio-service-mesh
```

File: quickstarts/istio/config.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+# Displayed in the UI (required)
+title: Istio Service Quickstart
```
Comment (jpvajda) [inline quickstarts/istio/config.yml:5]:
```suggestion
title: Istio Service Mesh
```

File: quickstarts/istio/config.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  Prometheus OpenMetrics Integration. [POMI](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic)
+
+  [Istio Standard Metrics](https://istio.io/latest/docs/reference/config/metrics/)
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  A dashboard view of services running in your Kubernetes cluster using an Istio Service Mesh.
+
+# Support level: New Relic | Verified | Community (required)
+level: New Relic
```
Comment (jpvajda) [inline quickstarts/istio/config.yml:23]: Is this officially supported by New Relic? Does it match our level criteria for support?

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#support-levels

**Review by jpvajda — CHANGES_REQUESTED** [review]
Thanks for this contribution! I had some feedback for you.

File: quickstarts/istio/config.yml (hunk 3 above)
Comment (shawnj) [reply quickstarts/istio/config.yml:23]: Right now New Relic is the only one supporting this dashboard.

**Comment by shawnj** [comment]
@jpvajda I removed the "Quickstart" reference from the title and id.  I just left `Istio-Service` as this dashboard doesn't necessarily apply to the whole mesh but to the services/applications running within the service mesh. The dashboards for Istio Mesh and Control Plane are coming later.

**Comment by jpvajda** [comment]
@shawnj Thanks! this looks good, one more change you don't need the (icon image)[quickstarts/istio/images/icon.jpeg](https://github.com/newrelic/newrelic-quickstarts/pull/976/files#diff-c53ddf39dc786d01860793437103b4bbd758091a4e5f5e075bf203de1f72db90)  we  still use the logo image you provided for the UI.

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#logos--icons

**Comment by shawnj** [comment]
Thanks @jpvajda .  Removed the image icon as requested.

**Review by jpvajda — APPROVED** [review]

**Pull request merged**